app/src/androidTest/java/at/bitfire/icsdroid/ui/AddCalendarEnterUrlFragmentValidationTest.kt

```diff
@@ -18,7 +20,7 @@ import org.junit.BeforeClass
 import org.junit.Rule
 import org.junit.Test
 
-class AddCalendarValidationFragmentTest {
+class AddCalendarEnterUrlFragmentValidationTest {
```

Comment:
Since this is really only testing `ValidationModel` maybe we can rename it to `ValidationModelTest`.
And maybe move `ValidationModel` and `ValidationModelTest` from their ui packages to the respective model packages each?

Reply:
Yup, just moved it 😉